tsconfig.json

```diff
@@ -1,11 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
     "module": "esnext",
     "declaration": true,
+    "types": [],
     "esModuleInterop": true,
     "moduleResolution": "Node",
-    "lib": ["es5", "es2015.collection"],
+    "lib": ["es2020"],
```

Comment:
Pulling changes from the main components config

1. Bump target and lib to es2015/es2020 respectively
2. Set `types: []`. The typescript default is "load everything from the `node_modules/@types` directory" which we do not want. We only want the types listed in the `lib` section